### PR TITLE
VTCCA-4509 Update Appoint an agent to one or more properties sortable table columns with accessible component

### PR DESCRIPTION
**Pull request by reddy-yattapu**
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4509

### CODE REVIEW

File: app/views/includes/designSystemsStylesheets.scala.html [hunk 1]
```diff
@@ -19,3 +19,4 @@
  <!--[if gt IE 8]><!--><link href='@routes.Assets.at("stylesheets/application.css")' media="screen" rel="stylesheet" type="text/css" /><!--<![endif]-->
  <link href="@routes.Assets.at("stylesheets/searchAndSort.css")" media="screen" rel="stylesheet" type="text/css" />
  <link href="@routes.Assets.at("stylesheets/search.css")" media="screen" rel="stylesheet" type="text/css" />
+<link href="@routes.Assets.at("stylesheets/search.css")" media="screen" rel="stylesheet" type="text/css" />
```
Comment (zahiriqbal) [inline app/views/includes/designSystemsStylesheets.scala.html:22]: Duplicate line?

File: app/views/propertyrepresentation/revokeAgentProperties.scala.html [hunk 2]
```diff
@@ -41,7 +41,7 @@
 }
 
 @addressColumnHeaderLink = {
-    <a href="@agentAppointment.routes.AppointAgentController.sortRevokePropertiesByAddress(pagination, agentCode)" class="govuk-link" role="link" style="color: black">
+    <a href="@agentAppointment.routes.AppointAgentController.sortRevokePropertiesByAddress(pagination, agentCode)" role="link">
         @messages(s"propertyRepresentation.th.${ADDRESS.toString}")
```
Comment (zahiriqbal) [inline app/views/propertyrepresentation/revokeAgentProperties.scala.html:45]: Do you have to camel case the address: @messages(s"propertyRepresentation.th.${ADDRESS.toString}")?

File: app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html [hunk 3]
```diff
@@ -65,19 +65,15 @@
 @addressColumnHeaderLink = {
     <a id="sort-by-address"
        href="@agentAppointment.routes.AppointAgentController.sortPropertiesForAppoint(ADDRESS, pagination, agentCode, agentAppointed = appointedAgentToggle, backLinkAsString).url"
-       class="govuk-link"
-       role="link"
-       style="color: black">
+       role="link">
```
Comment (darrell-plant) [inline app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html:68]: do we not need `class="govuk-link"`?
do we need/use `role="link"`?

File: app/views/propertyrepresentation/revokeAgentProperties.scala.html (hunk 2 above)
Comment (darrell-plant) [reply app/views/propertyrepresentation/revokeAgentProperties.scala.html:45]: good question

File: app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html (hunk 3 above)
Comment (reddy-yattapu) [reply app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html:68]: We don't need class="govuk-link" for sortable header columns as styles for sortable headers declared in _sortable.scss

I kept role="link" as prototype and other places declared this attribute, may be used for other purpose but not sure where it is being used

File: app/views/includes/designSystemsStylesheets.scala.html (hunk 1 above)
Comment (reddy-yattapu) [reply app/views/includes/designSystemsStylesheets.scala.html:22]: Removed duplicate include

File: app/views/propertyrepresentation/revokeAgentProperties.scala.html (hunk 2 above)
Comment (reddy-yattapu) [reply app/views/propertyrepresentation/revokeAgentProperties.scala.html:45]: https://github.com/hmrc/voa-property-linking-frontend/blob/main/conf/messages#L568 value of the ADDRESS key is camel case
By the way I did not tocuhed  any message key/values as nothing to do with scope of the story

Comment (darrell-plant) [reply app/views/propertyrepresentation/revokeAgentProperties.scala.html:45]: sorry, I misread that.

**Comment by darrell-plant** [comment]
LGTDz 👍

**Pull request merged**